Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp

```diff
@@ -321,6 +321,70 @@ void CAntiAim::Run(CUserCmd* pCmd, bool* pSendPacket) {
 					pCmd->viewangles.y = lastFakeAngle;
 					break;
 				}
+			case 9:
+				{
+				static int Timer = 0;
+				if (*pSendPacket == bSendReal == false)
+				if (Timer++ >= 30)
+				{
+			       pCmd->viewangles.y = 135.f;
+				   Timer = 0;
+				   break;
+				   //this
+				}
+				else if (*pSendPacket == bSendReal == false)
+				{
+				   pCmd->viewangles.y = 0.f;
+				   //static fake
+				   break;
+				}
+			   else if (pCmd->viewangles.y == 0.f)
+			   {
+				  *pSendPacket == bSendReal == true;
+			   }
+			   else 
+			   {
+				  *pSendPacket == bSendReal == false;
```

Comment:
bruh what does this meannnnnnn what does it do i ddont get it

Reply:
IT SENDS PACKET BACK